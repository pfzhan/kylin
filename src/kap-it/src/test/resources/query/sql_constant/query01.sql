select cast('112.56' as tinyint), cast('112.56' as smallint), cast('112.56' as int), cast('112.56' as integer)