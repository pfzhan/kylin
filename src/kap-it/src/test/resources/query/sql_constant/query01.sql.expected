select cast('112.56' as tinyint) as cast1, cast('112.56' as smallint) as cast2, cast('112.56' as int) as cast3, cast('112.56' as integer) as cast4, cast('112.56' as bigint) as cast5